roles/installer/molecule/default/prepare.yml

```diff
@@ -8,3 +8,9 @@
   roles:
     - puppet_repositories
     - foreman_repositories
+  tasks:
+    - name: Ensure langpacks on EL8
```

Comment:
can you add the same hack to the `foreman_proxy_certs_generate` role please?

Reply:
Done! :rocket: